### PR TITLE
Supply converter digiline

### PR DESCRIPTION
**Pull request by DS-Minetest**
related: #168

### CODE REVIEW

File: technic/machines/supply_converter.lua [hunk 1]
```diff
@@ -58,6 +65,48 @@ local mesecons = {
...
+					mesecon_mode = meta:get_int("mesecon_mode")
+				})
+				return
+			elseif msg == "off" then
+				meta:set_int("enabled", 0)
+			elseif msg == "on" then
+				meta:set_int("enabled", 1)
+			elseif msg == "toggle" then
+				local onn = meta:get_int("enabled")
+				onn = -(onn-1) -- Mirror onn with pivot 0.5, so switch between 1 and 0.
```
Comment (kaeza) [inline technic/machines/supply_converter.lua]: I think `onn = onn==0 and 1 or 0` would be more readable here.

**Comment by kaeza** [comment]
If nobody has objections I would like to merge this (will edit that code locally).

@VanessaE, @Ekdohibs ?

File: technic/machines/supply_converter.lua (hunk 1 above)
Comment (Ekdohibs) [inline technic/machines/supply_converter.lua]: either that or `1-onn` which is quite standard and readable as well

**Comment by kaeza** [comment]
Merged as 6c4e90d. I didn't touch the code. If somebody is bothered enough please change it.

Thank you!

File: technic/machines/supply_converter.lua (hunk 1 above)
Comment (DS-Minetest) [inline technic/machines/supply_converter.lua]: Huh, I thought, I had changed it to `1-onn`. Perhaps it was in the other PR. :thinking:

Comment (asl97) [inline technic/machines/supply_converter.lua]: @DS-Minetest you changed it for the forcefield only in #338@[1c2f30f6](https://github.com/minetest-technic/technic/pull/338/commits/1c2f30f67b932046ce7000fb44ffb78e385d3a03#diff-ed22d5b00293f6bd0ca9de5f80a3a469R184)

**Pull request closed without merging**